docs/conf.py

```diff
@@ -63,6 +64,10 @@
 ]
 templates_path = ['_templates']
 
+intersphinx_mapping = {
+    "qiskit": ("https://qiskit.org/documentation/", None),
+}
+
```

Comment:
This stuff is just so I could make the links to various Qiskit objects work correctly.